katex.gemspec

```diff
@@ -24,7 +24,7 @@ Gem::Specification.new do |s|
 
   s.add_dependency 'execjs', '~> 2.7'
 
-  s.add_development_dependency 'bundler', '~> 2.0.1'
+  s.add_development_dependency 'bundler', '~> 2.0'
```

Comment:
While not strictly mandatory, pinning bundler to a specific minor version seemed overly restrictive, but I can undo this change.